src/Selfie.pas

```diff
@@ -0,0 +1,170 @@
+unit Selfie;
+
+interface
+
+uses
+  Math,
+  D3DX9, 
+  Direct3D9,
+  Typestuff,
+  muksoka;
+
+type TSelfie = class
+public
+  isSelfieModeOn: boolean;
+  zoomlevel: (CLOSE, NORMAL, FAR);
```

Comment:
FAR helyett majd valami mas kene mert az keyword.

Reply:
Kulon typeba kirakni se lenen utolso gondolat.

Reply:
Renamed to WIDE